src/switch_core_media.c

```diff
@@ -10631,59 +10644,62 @@ SWITCH_DECLARE(switch_status_t) switch_core_media_activate_rtp(switch_core_sessi
-					switch_port_t remote_port = v_engine->remote_rtcp_port;
+				if (switch_channel_var_true(smh->session->channel, "telnyx_disable_rtcp") ||
+					!switch_channel_get_variable(smh->session->channel, "telnyx_disable_rtcp")) {
+					if ((val = switch_channel_get_variable(session->channel, "rtcp_video_interval_msec")) || (val = smh->mparams->rtcp_video_interval_msec)) {
+						const char *rport = switch_channel_get_variable(session->channel, "rtp_remote_video_rtcp_port");
+						switch_port_t remote_port = v_engine->remote_rtcp_port;
 
-					if (rport) {
-						remote_port = (switch_port_t)atoi(rport);
-					}
-					if (!strcasecmp(val, "passthru")) {
-						switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_INFO, "Activating VIDEO RTCP PASSTHRU PORT %d\n", remote_port);
-						switch_rtp_activate_rtcp(v_engine->rtp_session, -1, remote_port, v_engine->rtcp_mux > 0);
-					} else {
-						int interval = atoi(val);
-						if (interval < 100 || interval > 500000) {
-							switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_ERROR,
-											  "Invalid rtcp interval spec [%d] must be between 100 and 500000\n", interval);
-							interval = 5000;
+						if (rport) {
+							remote_port = (switch_port_t)atoi(rport);
 						}
-						switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_INFO,
-										  "Activating VIDEO RTCP PORT %d interval %d mux %d\n", remote_port, interval, v_engine->rtcp_mux);
-						switch_rtp_activate_rtcp(v_engine->rtp_session, interval, remote_port, v_engine->rtcp_mux > 0);
+						if (!strcasecmp(val, "passthru")) {
+							switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_INFO, "Activating VIDEO RTCP PASSTHRU PORT %d\n", remote_port);
+							switch_rtp_activate_rtcp(v_engine->rtp_session, -1, remote_port, v_engine->rtcp_mux > 0);
+						} else {
+							int interval = atoi(val);
+							if (interval < 100 || interval > 500000) {
+								switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_ERROR,
+												  "Invalid rtcp interval spec [%d] must be between 100 and 500000\n", interval);
+								interval = 5000;
+							}
+							switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_INFO,
+											  "Activating VIDEO RTCP PORT %d interval %d mux %d\n", remote_port, interval, v_engine->rtcp_mux);
+							switch_rtp_activate_rtcp(v_engine->rtp_session, interval, remote_port, v_engine->rtcp_mux > 0);
 
-					}
+						}
 
 
-					if (v_engine->ice_in.cands[v_engine->ice_in.chosen[1]][1].ready && v_engine->ice_in.cands[v_engine->ice_in.chosen[0]][0].ready &&
-						!zstr(v_engine->ice_in.cands[v_engine->ice_in.chosen[1]][1].con_addr) && 
-						!zstr(v_engine->ice_in.cands[v_engine->ice_in.chosen[0]][0].con_addr)) {
+						if (v_engine->ice_in.cands[v_engine->ice_in.chosen[1]][1].ready && v_engine->ice_in.cands[v_engine->ice_in.chosen[0]][0].ready &&
```

Comment:
Same here...

This should not be commited. Can you please clean this up @StefanYohansson (maybe you need to checkout the file and re-apply your changes only to relevant part - around lines 4850 - 4880